README.md

```diff
@@ -100,11 +100,79 @@ this can be used with `watch`.
 watch -c juju status --color
 ```
 
-### Alternate deployment methods
+## Alternate deployment methods
 
+### Deploying storage
```

Comment:
I think I agree with this. Couple alternatives come to mind:

1. Move it to the bottom of the Usage section (with a note that it's optional)
2. Make it a new top-level section
3. Move it within "Scale out Usage"? I dunno, it's kinda like scaling storage?

To prevent "Usage" from getting too big, and because I'm not sure if "Scale out Usage" is a good fit: I think I prefer option 2. Make it top level. But I have no strong feelings one way or the other.